Xamarin.Forms.Controls/GalleryPages/CollectionViewGalleries/CarouselViewGalleries/CarouselViewGallery.cs

```diff
@@ -14,8 +14,8 @@ public CarouselViewGallery()
 
 			var button = new Button
 			{
-				Text = "Enable CarouselView",
-				AutomationId = "EnableCarouselView"
+				Text = "Enable IndicatorView",
```

Comment:
Since CarouselView will be behind the flag a bit longer, do we need to keep them both for now?